src/styles/quiz-card/quiz-card.css

```
@@ -0,0 +1,47 @@
+.quiz__general {
```

This is the class for the quiz-card `section` element, right? So this is the block of BEM and then it would be:
```suggestion
.quiz-card {
```